fission-web-server/test/Fission/Test/Web/Server/Root.hs

```diff
@@ -0,0 +1,27 @@
+module Fission.Test.Web.Server.Root (spec) where
+
+import           Servant
+
+import qualified Fission.Web.API.Types           as API
+
+import           Fission.Test.Web.Server.Prelude
+
+spec :: Spec
+spec =
+  describe "GET /" do
+    with rootServer do
+      it "is always be successful" do
```

Comment:
```suggestion
      it "is always successful" do
```

:stuck_out_tongue:

Reply:
haha your English is better than mine today